tools/flash_rx_via_ap_passthru3.py

```diff
@@ -334,6 +339,9 @@ def mlrs_flash_firmware_file(uart, baud, file, full_erase_flag):
     comport_list = []
     for com in sorted(coms):
         comport_list.append(com.device)
+    if not comport_list:
+        comport_list.append('None Found')
+        print('No serial ports found.  Please connect FC')
```

Comment:
"ERROR: No serial ports found. Please connect your FC."

Reply:
OK.  If you like that better.  The main thing is not to crash which is what it was doing...

Reply:
yeah, it's just nitpick. I try to be somewhat consistent in the output across all py scripts in the mlrs repo.